src/layouts/Course__c-HEDA Course Layout.layout

```diff
@@ -16,7 +16,7 @@
             </layoutItems>
             <layoutItems>
                 <behavior>Edit</behavior>
-                <field>Description__c</field>
+                <field>Extended_Description__c</field>
```

Comment:
Actually yes, I was instructed to have the old field removed from the page layout by @wongmike. This is the reason behind the batch process, so admins of existing Orgs can move the data over after updating their existing layout.